app/main.py

```diff
@@ -1,2 +1,47 @@
+from collections.abc import Hashable
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.hash_table: list = [None] * self.capacity
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.length == round(self.capacity * self.load_factor):
+            self.resize()
+        table_index = hash(key) % self.capacity
+        if self.hash_table[table_index] is None:
+            self.hash_table[table_index] = []
+        for cell in self.hash_table[table_index]:
+            if cell[0] == key and cell[1] == hash(key):
+                cell[2] = value
+                return
+
+        self.hash_table[table_index].append([key, hash(key), value])
+        self.length += 1
+
+    def __getitem__(self, item: Any) -> Any:
```

Comment:
key have to be Hashable
```suggestion
    def __getitem__(self, item: Hashable) -> Any:
```